core/build.gradle

```diff
@@ -1,39 +1,20 @@
-apply plugin: 'com.android.library'
-apply plugin: "kotlin-android"
+apply plugin: "java-library"
+apply plugin: "kotlin"
 apply plugin: "com.github.dcendents.android-maven"
 apply plugin: "com.jfrog.bintray"
 
-project.ext.description = "core classes of the uPort android SDK"
-
-android {
-    compileSdkVersion compile_sdk_version
-
-    defaultConfig {
-        minSdkVersion min_sdk_version
-        targetSdkVersion target_sdk_version
-        versionCode 1
-        versionName "1.0"
-
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-    }
-
-}
```

Comment:
I removed them. The core library is no longer an AAR, it will be delivered as JAR.

It had no android dependencies and this makes it easier to use in unconventional contexts.
Other modules will follow in future PRs, because they don't have android dependencies either.